docs/dev/dependencies.md

```diff
@@ -0,0 +1,22 @@
+# Packaging
+
+The packaging is handled by the `pyproject.toml` file for python dependencies,
+and by `package.json` for JavaScript dependencies.
+
+## Python
+
+Python dependencies are pinned, and we're relying on Github's dependabot to
+update them for us, via pull requests.
+
+## JavaScript
+
+Dependencies are **not** pinned, but a `^` character is used instead ([defined
+by node semver](https://github.com/npm/node-semver#caret-ranges-123-025-004)),
+meaning the next minor or patch versions will be installed (but not the next major)
+
+The installed libs needs to be:
+
+- Added to the `package.json`
+- Added in the `scripts/vendorsjs.sh` script
+- Loaded in the HTML templates.
```

Comment:
+ Added in `umap/test/index.html`
+ `make vendors`

?

Reply:
> Added in umap/test/index.html

With the modules it's not a requirement anymore, so it really depends what you want to do.

`make vendors` indeed!

Reply:
(updated on master :see_no_evil:)

Reply:
> make installjs vendors

Is there an use case for running `make vendors` without running `installjs` first? (I'm wondering if we can make one a dependency of the other)

Reply:
I don't see any use case to run only one but no the other, let's do that :)